lib/AppInfo/Application.php

```diff
@@ -29,4 +30,29 @@ public function __construct(array $urlParams = []) {
 		parent::__construct('spreed', $urlParams);
 	}
 
+	public function registerHooks() {
+		$notifier = $this->getContainer()->query(BackendNotifier::class);
+
+		$dispatcher = $this->getContainer()->getServer()->getEventDispatcher();
+		$dispatcher->addListener('\OCA\Spreed\Room::postAddParticipants', function($event) use ($notifier) {
```

Comment:
I'd use `->addListener(Room::class . '::postAddParticipants')` like we do in other places as well already

Reply:
Changed. I based my code on the `dav` app, where the complete strings are used - using the classes makes more sense though!